docs/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "jsx": "react",
+    "outDir": "dist",
     "rootDir": "src",
-    "outDir": "dist"
+    "target": "es5"
```

Comment:
Tell me about how you came to this "target" option.

Reply:
looking at the gatsby docs